bin/webpack-dev-server

```diff
@@ -23,7 +23,7 @@ end
 begin
   dev_server = YAML.load_file(CONFIG_FILE)["development"]["dev_server"]
 
-  DEV_SERVER_HOST = "http#{"s" if args('--https') || dev_server["https"]}://#{args('--host') || dev_server["host"]}:#{args('--port') || dev_server["port"]}"
+  DEV_SERVER_HOST = "http#{"s" if args('--https') || dev_server["https"]}://#{dev_server["host"]}:#{args('--port') || dev_server["port"]}"
```

Comment:
This variable is used to figure out how the webpage should access Webpack, and thus should use `dev_server["host"]`. But `--host 0.0.0.0` is still passed in to the `webpack` command below (via `Procfile.dev`) and so Webpack serves from 0.0.0.0.